k8s/namespaces/private-law/prl-citizen-frontend/demo.yaml

```diff
@@ -14,7 +14,7 @@ spec:
     path: stable/prl-citizen-frontend
   values:
     java:
-      image: hmctspublic.azurecr.io/prl/cos:latest #{"$imagepolicy": "flux-system:prl-citizen-frontend"}
+      image: hmctspublic.azurecr.io/prl/cos:latest #{"$imagepolicy": "flux-system:demo-prl-citizen-frontend"}
```

Comment:
Can you use a specific image here if possible

Reply:
Updated with PR image.